manifests/init.pp

```diff
@@ -85,6 +88,7 @@
   String $rvm_ruby = '2.7',
   String $rvm_branch = 'stable',
   Optional[String] $scl_ruby = $katello_devel::params::scl_ruby,
+  Optional[String] $modulestream_nodejs = $katello_devel::params::modulestream_nodejs,
```

Comment:
I've tested and confirmed this works on EL8 with node version 12 and 14. Therefore I should probably change this type to an `Enum` when I have some time later.

BTW, regarding the `npm install` timeout observed in CI, on my local machine it took ~32 minutes in each test case. The default value probably needs to be updated here as well.